frontend/scss/state/_s-sticky-sidebar.scss

```diff
@@ -145,5 +145,34 @@ For _p-issuearticle-show.scss and _p-issue-show.scss
         bottom: 0;
       }
     }
+
+    .m-sidebar-toggle {
+      display: none;
+    }
+  }
+
+  @include breakpoint('xsmall') {
+    .m-sidebar-toggle--publication__logo {
+      margin-bottom: 40px;
+      white-space: nowrap;
+
+      a:hover {
+        text-decoration: underline;
+      }
+    }
+
+    &.is-sidebar-fixed.s-header-hide:not(.s-scroll-direction-up) {
+      .m-sidebar-toggle {
+        opacity: 1;
+        position: fixed;
+        top: 0;
+        z-index: map-get($zindexs, 'header');
+        &::before {
+          left: 0;
+          margin-left: -4.6875vw;
+          margin-right: -4.6875vw;
+        }
+      }
+    }
```

Comment:
All this is to make the TOC play nice with the main menu: when the main menu is hidden and you are not scrolling up, fix the TOC to the top of the page.